conversions/celsius_to_fahrenheit.py

```diff
@@ -1,34 +1,28 @@
 """ Convert temperature from Celsius to Fahrenheit """
 
 
-def celsius_to_fahrenheit(celsius):
+def celsius_to_fahrenheit(celsius: float) -> float:
     """
-    Convert a given value from Celsius to Fahrenheit
+    Convert a given value from Celsius to Fahrenheit and round it to 2 decimal places.
 
-    >>> celsius_to_fahrenheit(-40)
+    >>> print(celsius_to_fahrenheit(-40))
     -40.0
-    >>> celsius_to_fahrenheit(-20)
+    >>> print(celsius_to_fahrenheit(-20))
     -4.0
-    >>> celsius_to_fahrenheit(0)
+    >>> print(celsius_to_fahrenheit(0))
     32.0
-    >>> celsius_to_fahrenheit(20)
+    >>> print(celsius_to_fahrenheit(20))
     68.0
-    >>> celsius_to_fahrenheit(40)
+    >>> print(celsius_to_fahrenheit(40))
```

Comment:
```suggestion
    >>> print(celsius_to_fahrenheit("40"))
```
Just to demonstrate and test that we do take properly formed strings.

Also, please add a floating point input test.